package excaptionHandlingPackage;

public class NestedTryExample {

	public static void main(String[] args) {
	
		try{
			try{
				System.out.println("Inside nested try block");
				int a=300/0;
			}catch(ArithmeticException e){
				System.out.println(e);
			}
			try{
				int b [] ={13,15,17,19,21,23,24};
				System.out.println(b[12]);	
			}catch(Exception e){
				System.out.println(e);
			}
			System.out.println("Continue");
		}	

	}

}
